_docs/Backend/status.md

```diff
@@ -13,4 +13,4 @@ order: 4
 ### Next Release
 
 The Android app has a 1 month release cycle. The next scheduled release is
```

Comment:
Why not just make it generic something like `The Android app has a 1 month release cycle on the 20th of each month.`